### DISCUSSION TITLE
Is there any way to set the cursor position by screen coordinates?

### DISCUSSION THREAD

**Discussion by chshouyu** [ueberdosis/tiptap#1499] [General Questions] [answered]
My editor has a function which can toggle editor mode and html preview mode, like this:

![2021-06-22 00 33 47](https://user-images.githubusercontent.com/1304342/122797651-47981580-d2f2-11eb-9d4d-e09546eaa13f.gif)

I have read the docs, and found the `focus()` or `setTextSelection ()` methods, but they all accept number, so, is there any official way to achieve this? thanks🙏

**Comment by philippkuehn** [comment, marked as answer]
posAtCoords is the thing you want: https://prosemirror.net/docs/ref/#view.EditorView.posAtCoords

**Reply by chshouyu** [reply]
Wow, thanks for the big help, love you❤️
